src/pages/HomePage.vue

```diff
@@ -19,9 +18,6 @@ const HomePage = {
     HeroSection,
     FooterSection,
   },
-  mounted() {
-    this.$store.commit(RESET_QUERY);
```

Comment:
You can't remove this because this is resets the query when the user navigates back to the home page. This is to clear any previous query filters that the user set before, so that a new search doesn't have any filter applied

Reply:
@brenoferreira you can kindly take a look at the changes i just made